ext/drreg/drreg.c

```diff
@@ -828,6 +828,9 @@ drreg_forward_analysis(void *drcontext, instr_t *start)
         aflags_new = instr_get_arith_flags(inst, DR_QUERY_INCLUDE_COND_SRCS);
         /* reading and writing counts only as reading */
         aflags_new &= (~(EFLAGS_READ_TO_WRITE(aflags_new)));
+        /* also checks aflags_cur to make sure the writes are 
```

Comment:
comment nit: I guess this comment can be summarised as `writing doesn't count if already read`, like the other comments here.

Reply:
Thanks! I have fixed the long comments with the summarized ones.